staging/src/k8s.io/code-generator/cmd/informer-gen/generators/groupinterface.go

```diff
@@ -98,18 +99,19 @@ type Interface interface {
 }
 
 type group struct {
-	$.interfacesSharedInformerFactory|raw$
+	factory $.interfacesSharedInformerFactory|raw$
+	filter  $.interfacesFilterFunc|raw$
 }
 
 // New returns a new Interface.
-func New(f $.interfacesSharedInformerFactory|raw$) Interface {
-	return &group{f}
+func New(f $.interfacesSharedInformerFactory|raw$, filter $.interfacesFilterFunc|raw$) Interface {
```

Comment:
@ncdc Also worth noting that this changes the `New` constructor for the group and version interface as well, in case anybody is depending upon these directly (although I doubt they are, given it already accepts a SharedInformerFactory as an argument).

Reply:
Yeah, I opted not to comment on this one since it really only exists as an intermediary 😄 .